Orange/widgets/unsupervised/owmds.py

```diff
@@ -323,8 +324,7 @@ def _initialize(self):
         self.openContext(self.data)
         self._invalidated = not (matrix_existed and
                                  self.effective_matrix is not None and
-                                 np.array_equal(effective_matrix,
-                                                self.effective_matrix))
+                                 array_equal(effective_matrix, self.effective_matrix))
```

Comment:
Even so, I'd prefer to use this function here. The reason is simple: when I forget how to check if the data table has changed (because I usually always assume we check something with domains and such), I look at how it's done in existing widgets. Having this function used as much as possible will make it less likely somebody will go and implement their own way of doing it (like I did).